### ISSUE TITLE
tcp listen alarm integer expression expected

### ISSUE THREAD

**Issue by Tahvok** [firehol/netdata#3733]
I have some hosts that has a known issue of packet loss.

Looking at error log, I've found this:
```
2018-05-18 05:19:48: alarm-notify.sh: INFO: sent telegram notification for: ecprdmctl03-opens ipv4.tcplistenissues.1m_ipv4_tcp_listen_drops is CRITICAL to '-248101019'
/usr/libexec/netdata/plugins.d/alarm-notify.sh: line 1556: [: NO: integer expression expected
```

Not sure what's going on here, and why there is a negative number there

edit: wrong investigation. Updated the issue accordingly.

**Comment by ktsaou** [comment]
Thank you for reporting this!
It is a bug. Fixed it in #3753

**Comment by ktsaou** [comment]
merged it.
thank you for reporting this!
